lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.ThermalZone.rb

```diff
@@ -0,0 +1,152 @@
+class ASHRAE901PRM < Standard
+  # @!group ThermalZone
+
+  # Determine the fan power limitation pressure drop adjustment
+  # Per Table 6.5.3.1-2 (90.1-2019)
+  #
+  # @param themal_zone
+  def thermal_zone_get_fan_power_limitations(thermal_zone, is_energy_recovery_required)
+    fan_pwr_adjustment_in_wc = 0
+    # Get autosized zone design supply air flow rate
+    dsn_zone_air_flow_m3_per_s = thermal_zone.designAirFlowRate.to_f
+    dsn_zone_air_flow_cfm = OpenStudio.convert(dsn_zone_air_flow_m3_per_s, 'm^3/s', 'cfm').get
+
+    # Retrieve credits from zone additional features
+    # Fully ducted return and/or exhaust air systems
+    if thermal_zone.additionalProperties.hasFeature('has_fan_power_credit_fully_ducted')
+      mult = thermal_zone.additionalProperties.getFeatureAsDouble('has_fan_power_credit_fully_ducted').to_f
```

Comment:
Would mult be "1" if a single system serves the zone, and "2" if there is a DOAS?

Reply:
Yes.